### PR TITLE
Updated README and compressed large file

### PR DESCRIPTION
**Pull request by LeonorLunatech**

### CODE REVIEW

**Comment by VincentBrule** [comment]
Can you resize the image, we don't need that much pixels (6800x4800) ;)

File: README.md [hunk 1]
```diff
@@ -35,6 +35,18 @@ and delete this line:
 
 then run brew install giter8 again.
 
+# Provide compressed images
+Every blogpost has at least one image which makes the Lunatech blog relatively heavy in memory consumption.
+One way of compressing images is by using `pngcrush`. You can install it using brew:
```
Comment (j-madden) [inline README.md:40]: Might be nice to add a follow-up mention that compressing combats this memory consumption.

One way of to combat this is by compressing images via [pngcrush](https://pmt.sourceforge.io/pngcrush/).
Installing it as follows:

**Comment by LeonorLunatech** [comment]
> Can you resize the image, we don't need that much pixels (6800x4800) ;)

I guess you can crop it a bit, but it's not my image not blogpost so I decided to no mess with it and just compress it

File: README.md (hunk 1 above)
Comment (LeonorLunatech) [reply README.md:40]: Good point

**Review by j-madden — APPROVED** [review]

**Pull request merged**